[FLOSS H3] Fixing deadlinks on the Website in the Availability Section (#4770)
ElektraInitiative/libelektra: doc/news/_preparation_next_release.md

- Remove duplicated link to `TESTING.md` file _(@stefnotch)_, _(@janldeboer)_
- <<TODO>>
- Restructure parts of the links on the website _(@stefnotch)_, _(@janldeboer)_

> - Fixed broken Links with a link to the Mainpage _(@Dynamichost96)_

Reviewer: ```suggestion
- Removed broken links to packages for Linux distributions. _(@Dynamichost96)_
```

Reviewer: The text in the release notes has to be updated to correctly describe the actual changes in the PR.

Author: Thanks, i changed the release note.
